packages/react-components/react-breadcrumb-preview/stories/Breadcrumb/BreadcrumbFocusMode.stories.tsx

```diff
@@ -32,54 +30,27 @@ const buttonItems: Item[] = [
   {
     key: 2,
     item: 'Item 2',
-    buttonProps: {
-      icon: <GridDots20Regular />,
-    },
   },
   {
     key: 3,
     item: 'Item 3',
   },
-  {
-    key: 4,
-    item: 'Item 4',
-    buttonProps: {
-      icon: <CalendarMonth />,
-    },
-  },
-  {
-    key: 5,
-    item: 'Item 5',
-  },
 ];
 
 function renderButton(el: Item, isLastItem: boolean = false) {
   return (
     <React.Fragment key={`${el.key}-button`}>
       <BreadcrumbItem>
-        <BreadcrumbButton
-          {...el.buttonProps}
-          current={isLastItem}
-          onClick={isLastItem ? undefined : el.buttonProps?.onClick}
-        >
+        <BreadcrumbButton {...el.buttonProps} current={isLastItem}>
           {el.item}
         </BreadcrumbButton>
       </BreadcrumbItem>
       {!isLastItem && <BreadcrumbDivider />}
     </React.Fragment>
   );
 }
-export const BreadcrumbFocusMode = () => (
-  <>
-    <h2>`tab` - default</h2>
-    Navigation using `tab` key.
-    <Breadcrumb aria-label="Breadcrumb with `tab` focusMode">
-      {buttonItems.map(el => renderButton(el, el.key === buttonItems.length - 1))}
-    </Breadcrumb>
-    <h2>`arrow`</h2>
-    Navigation using arrow keys
-    <Breadcrumb aria-label="Breadcrumb with `arrow` focusMode" focusMode="arrow">
-      {buttonItems.map(el => renderButton(el, el.key === buttonItems.length - 1))}
-    </Breadcrumb>
-  </>
+export const FocusModeArrow = () => (
```

Comment:
Add a description in the metadata for this story.

Reply:
done